cli/waiter/data_format.py

```diff
@@ -158,6 +159,36 @@ def load_data(options):
         if not content:
             raise Exception(f'Unable to load {input_format} from {input_file}.')
 
+        context_dict = {}
+        context_provided = False
+        context_file = options.get('context_file')
+        if context_file:
+            context_provided = True
+            logging.debug(f'reading context as yaml from {context_file}')
+            context_content = load_file(context_file)
+            if not context_content:
+                raise Exception(f'Unable to load yaml from {context_file}.')
```

Comment:
Is the format always yaml? Can we make the error message: "Unable to load yaml or json from..."

Reply:
Fixed messaging. The context file is parsed as yaml, yaml being a superset of json also parses json files :)